highfidelity/hifi
# Interface should always start in first person WL 21423

### To test prevention of Mirror mode:
1. Open Interface.
2. If camera is not in mirror mode, set camera to mirror mode.
3. Close and reopen Interface.
4. Camera should begin in first-person mode instead of mirror mode.

### To test saving/reloading of third-person mode:
1. Open Interface. Note whether the camera begins in first-person or third-person mode. Camera should not be in mirror mode (i.e. already pass the above test).
2. If camera began in first-person, switch to third-person. If camera began in third-person, switch to first-person.
3. Close and reopen Interface.
4. Camera should remain in the mode it was set to in step 2, and not revert to the mode noted in step 1.

### And to test that camera does not change modes without user action:
1. Open Interface. Note whether the camera begins in first-person or third-person mode. Camera should not be in mirror mode.
2. Close and reopen Interface.
3. Camera should remain in the mode it began in in step 1.
